src/styles/main.scss

```diff
@@ -6,9 +6,14 @@
 @import "card";
 @import "stars";
 
+// * {
```

Comment:
why did you remove this? if you want to have some custom box-sizing property, use it there

Reply:
fixed, I don't use it